ui/components/Modal.js

```diff
@@ -1,13 +1,6 @@
 import React from "react";
 import { Modal, Backdrop, Fade } from '@mui/core'
-import { styled } from "@mui/material/styles";
-import { useTheme } from "@mui/system";
 
-const CustomModal = styled(Modal)(({ theme }) => ({
```

Comment:
This was perfect, i dont know why you removed it 😅 . can you revert it ?
in fact, the only change that was needed was the naming change.

Reply:
I thought we didn't a custom modal 😅 so I just did it in the way we have right now, in the codebase. I'll revert.😅

Reply:
sure